Automattic/wp-calypso
# WP Blaze widget: Add dedicated flag for widget redesign

## Proposed Changes

1. Previously the widget redesign was reusing the flag for campaign dash redesign: `promote-post/redesign-i2`.
Now we want to release  the widget redesign ahead of campaign dash redesign, so I created another flag for itself: `promote-post/widget-i2`.

2. Also, fixed a bug where it's not honoring the flag set in URL e.g `?flags=promote-post/redesign-i2`. The bug was it's selecting the widget js URL dynamically based on the flag.

## Testing Instructions

Fire up Calypso, turn on/off the feature flag `promote-post/widget-i2` in development.js, and see if the widget loads the correct version respectively.


## Pre-merge Checklist
